.github/actions/javascript/authorChecklist/authorChecklist.ts

```diff
@@ -1,10 +1,11 @@
 import * as core from '@actions/core';
 import * as github from '@actions/github';
 import escapeRegExp from 'lodash/escapeRegExp';
-import CONST from '../../../libs/CONST';
 import GithubUtils from '../../../libs/GithubUtils';
 import newComponentCategory from './categories/newComponentCategory';
 
+const CONST = require('../../../libs/CONST');
```

Comment:
Do you have any log of a problem that you had with tests before updating this import?

Reply:
<img width="1021" alt="image" src="https://github.com/Expensify/App/assets/25230417/806a6e18-a495-408b-ac05-7a640c790d63">
<img width="753" alt="image" src="https://github.com/Expensify/App/assets/25230417/4224b518-0635-4b19-bc24-a7c168094716">
<img width="1234" alt="image" src="https://github.com/Expensify/App/assets/25230417/5c079693-68b7-49e1-a3e8-471ab55f642f">
<img width="1537" alt="image" src="https://github.com/Expensify/App/assets/25230417/c7d15d1f-96af-4406-b9e0-64de174f36e3">
and many more but all of them seems similar

Reply:
I was able to resolve tests problems after updating imports in these files 😅  Well, it looks like it will be easier to do that after github actions files are migrated, as well as GithubUtils.
![image](https://github.com/Expensify/App/assets/23176449/b7919461-8c37-4ea4-8cbe-8fc5a33c99bb)